internal/pkg/deploy/cloudformation/cloudformation_integration_test.go

```diff
@@ -0,0 +1,176 @@
+// +build integration
+
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cloudformation_test
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/deploy/cloudformation"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	awsCf "github.com/aws/aws-sdk-go/service/cloudformation"
```

Comment:
nit: can you change it to `awsCF` instead? https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Sure! Good catch